src/platform_impl/windows/window_state.rs

```diff
@@ -320,6 +321,24 @@ impl WindowFlags {
             }
         }
 
+        if diff.contains(WindowFlags::ALWAYS_ON_BOTTOM) {
+            unsafe {
+                SetWindowPos(
+                    window,
+                    match new.contains(WindowFlags::ALWAYS_ON_BOTTOM) {
+                        true => HWND_BOTTOM,
+                        false => HWND_NOTOPMOST,
```

Comment:
This won't work correctly when switching from `AlwaysBottom` to  `AlwaysTop` as the diff will contain both flags.
With the current implementation it will end up in `HWND_NOTOPMOST` (==`WindowLevel::Normal`).

Both cases should be handled in one if condition and we only have one call to `SetWindowPos` which resolves the tri-state.

Reply:
Good catch, let me think if [843fd86](https://github.com/rust-windowing/winit/pull/2534/commits/843fd865627ec01c96dfad004a3c2e4f5cf6ed70) is what you have in mind.